index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel ="stylesheet" href = "styles.css">
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+
+<header id="Header" class="outer-content-container">
+    <div class="inner-content-container" id="header2">
+    <nav class="header-content__navigation">
+        <ul>
+            <li> <a href="Header" >Case studies </a></li>
+            <li> <a href="Header" >Services </a></li>
+            <li> <a href="Header" >Contact us </a></li>
+        </ul>
+    </nav>
+    <div class ="header-content__cta">
+        <h1> Your website is an investment in the success of your business.</h1>
+        <h2>We are here to help you succeed.</h2>
+        <button> Let's work together </button>
+        </div>
+
+    <div class="header-squiggle">     </div>
+    </div>
+</header>
+
+
+
+<main id="main1" class="outer-content-container">
+    <div class="inner-content-container">
+
+    <section class="main-section1-section">
+        <article class="main-article1">
+            <p class= "main-article1__text1">
+                Rekupe is a digital agency in Los Angeles.</P>
+            <p class= "main-article1__text1">
+                We use strategic UX, SEO optimization and beautiful visual designs to help your business grow. </p>
+            <p class="main-article1__text1">
+                We see every project as a fresh opportunity, pushing ourselves to create new digital experiences.
+            </p>
+            <p class= "main-article1__text2">
+                Have a question? Click <a href="https://www.youtube.com">here</a> to contact us!
+            </p>
+        </article>
+    </section>
+
+     <section id="main2" class="outer-content-container">
+         <div class ="inner-content-container">
+             <article class="main-section2__article">
+                 <div class="main-section2__div">
+                     <h2> Rolling Stone </h2>
+                     <img src="/assets/squiggle_line.svg"  alt="squiggle" class="squiggle-main" >
```

Comment:
Dit pad werkt niet bij mij. Je zult óf "./assets" óf "assets" moeten gebruiken
```suggestion
                     <img src="./assets/squiggle_line.svg"  alt="squiggle" class="squiggle-main" >
```

```suggestion
                     <img src="assets/squiggle_line.svg"  alt="squiggle" class="squiggle-main" >
```